### ISSUE TITLE
Translated Side Menu fragment link is not working

### ISSUE THREAD

**Issue by matamatanot** [react-hook-form/documentation#381]
Working
https://react-hook-form.com/get-started#ReactWebVideoTutorial

Not Working
https://react-hook-form.com/jp/get-started#ReactWebVideoTutorial
https://react-hook-form.com/pt/get-started#ReactWebVideoTutorial
etc...

-----

I read a little bit of code. It seems to be caused by `esLinks`.
https://github.com/react-hook-form/documentation/blob/0db20b179ba423b7659a93b2779687632dc58b2c/src/components/GetStartedPage.tsx#L37

I modified part of it as follows and it worked. This is an example, not an abstraction.

https://github.com/react-hook-form/documentation/blob/0db20b179ba423b7659a93b2779687632dc58b2c/src/components/SideMenu.tsx#L73

```
let enLink =
              typeof enLinks[index] === "function"
                ? enLinks[index]().title
                : enLinks[index].title

enLink =
              enLink === "Quick start" ? "クイックスタート" : enLink
```

**Comment by pmaier983** [comment]
Great find matamatanot! Would you like to open up a PR to solve this bug? If not ill fix it in the near future.

**Comment by pmaier983** [comment]
I'll be taking a look into this in the next few days FYI

**Comment by bluebill1049** [comment]
Thanks @pmaier983 👍

**Comment by matamatanot** [comment]
Thanks!

**Comment by pmaier983** [comment]
This Issue was closed when PR #385 was merged.
